model/task/task.go

```diff
@@ -1288,7 +1288,7 @@ func SetTasksScheduledTime(tasks []Task, scheduledTime time.Time) error {
 	return nil
 }
 
-// GetTaskIdBetweenIds gets the task between two task given that they are
+// FindMidwayTask gets the task between two task given that they are
```

Comment:
this can now be unexported and we should update the comment on `FindMidwayTaskFromIds` to just be descriptive of what the pure functionality of the function is.